.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,22 @@
+---
+name: Bug report
+about: Create a bug report for rustfmt
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior. If possible, please add a link to your project or a minimal working example.
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Meta**
+
```

Comment:
Perhaps an inline comment with instructions that will be visible to the user when creating the issue, but won't take up space in the issue once created

```suggestion
<!-- `rustfmt --version` -->

```

Reply:
That's good :+1: